config-templates/content_indices.properties

```diff
@@ -1,4 +1,4 @@
 #
 #Tue Jun 18 15:45:09 BST 2019
-latest=849c7197c96625b651c6cab221ffe652d2d6765c
-live=849c7197c96625b651c6cab221ffe652d2d6765c
+latest=05bbbb74fd52cd0aea3a048a3c1e6c434ce5efaf
```

Comment:
Are these changes actually necessary? The file is inside a folder called `config-templates`, so I would imagine that these are just examples that you copy somewhere else, and then update with the right values.

If they are truly necessary, then the comment above should also be updated, as I'd imagine that that's not the right date and time anymore.

Reply:
I think it's a good idea to update this here, as the new endpoint uses a piece of content introduced in this SHA. If we leave it on the old SHA in the template then it essentially sets it up to fail (locally at least) for any dev who doesn't think to update it.  I will change the comment to reflect the date of the SHA